pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -85,11 +85,11 @@ func (q *graceTerminateRSList) flushList(handler func(rsToDelete *listItem) (boo
 	for name, rs := range q.list {
 		deleted, err := handler(rs)
 		if err != nil {
-			klog.Errorf("Try delete rs %q err: %v", name, err)
+			klog.ErrorS(err, "Try delete rs err", "name", name)
```

Comment:
What kind of name is it?

Reply:
It is real server name generated from realServer.String(), let me change it to "realServer", name